pkg/gce-cloud-provider/compute/gce.go

```diff
@@ -47,8 +48,27 @@ const (
 	regionURITemplate = "projects/%s/regions/%s"
 
 	replicaZoneURITemplateSingleZone = "projects/%s/zones/%s" // {gce.projectID}/zones/{disk.Zone}
+	versionV1                        = "v1"
+	versionBeta                      = "beta"
+	versionAlpha                     = "alpha"
+	googleEnv                        = "googleapis"
 )
 
+var computeVersionMap = map[string]map[string]map[string]string{
+	googleEnv: {
```

Comment:
What's the purpose of keying this by "googleapis" if there is only one entry in the map? IMO let's just keep the prod/staging map at the top level, and add in the extra level later if needed.